### PR TITLE
added the oci provider

### PR DESCRIPTION
**Pull request by ASatanicPickle**
added the Oracle Cloud Infrastructure provider to the project.

### CODE REVIEW

**Comment by josephholsten** [comment]
Assuming this changelog conflict gets merged, anything else need to be done to merge this?

**Comment by ASatanicPickle** [comment]
Hi man,

You can leave it for now, I will fix the conflict and I need to update the
OCI stuff anyways.

I'll take care of it tomorrow, and thank you for all your work on this
plugin, my team and I use it all the time.

Thanks,
Mark.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# 1.3.8
```
Comment (mauve) [inline CHANGELOG.md]: you can leave out this entry, I will write something when I make the release.

File: README.md [hunk 2]
```diff
@@ -25,7 +25,7 @@
 
 - Auto-completion support
   1. `Terraform` high level types `(variable, resource, data, module ...)` auto completion support.
-  2. `resource` types `(aws_dynamodb_table ...)` auto completion support for `aws, azure & google`
+  2. `resource` types `(aws_dynamodb_table ...)` auto completion support for `aws, azure, google & oci`
```
Comment (mauve) [inline README.md]:
```suggestion
  2. `resource` types `(aws_dynamodb_table ...)` auto completion support for `aws`, `azure`, `google` and `oci`
```

File: package.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
   "name": "terraform",
   "displayName": "Terraform",
   "description": "Syntax highlighting, linting, formatting, and validation for Hashicorp's Terraform",
-  "version": "1.3.7",
+  "version": "1.3.8",
```
Comment (mauve) [inline package.json]: leave it out, I will fix when I make the release after merging

**Comment by ASatanicPickle** [comment]
Hi,

I removed the change log file from the PR and updated the OCI provider data to the latest version.

Thanks,
Mark.

**Comment by mauve** [comment]
Thanks for your contribution, I have rebased and merged it outside of this PR (your name is still on the commit).

Will be released in a few minutes as 1.3.9

**Pull request closed without merging**